docs/index.rst

```diff
@@ -45,6 +45,17 @@ API Documentation
   asdf/user_api
   asdf/developer_api
 
+Developer Overview
+==================
+
+Currently a work in progress. Intended to give an overview of how the various 
+parts of ASDF interact and which modules do what and how.
+
+.. toctree:
```

Comment:
Needs a double colon, hopefully this will fix the error.

Reply:
I was seeing single when I should have been seeing double!